custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -85,6 +85,8 @@ def __init__(
             negative_class_label=negative_class_label,
             class_labels=class_labels,
         )
+        self._deployment = None
+        self._training_data_available = True
```

Comment:
Should we consider having default value `False` if most deployments may not have training data available? Or set this to `None` and then set it to be the bool of L147?

Reply:
Actually, its only for TextGen there is high probability training data not available - but for most of the other target types it should be available - hence keeping it True.  Or the code assumed training data present before this change and want to keep that behavior